### PR TITLE
Rename range to xrange

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Rename `range` to `xrange`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,7 +191,7 @@ nimble install pylib
 - [x] `oct()`
 - [x] `ord()`
 - [x] `print("foo")` / `print "foo"` Python2 like
-- [x] `range()`
+- [x] `range()` but named `xrange()` Python2 like
```
Comment (Yardanico) [inline README.md:194]:
```suggestion
- [x] `range()` but named `xrange()` like in Python 2
```

File: src/pylib/xrange.nim [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
     start, stop: T
     step, len: int
 
 func `$`*[T](rng: Range[T]): string =
   if rng.step != 1:
-    "range($1, $2, $3)".format(rng.start, rng.stop, rng.step)
+    "range(" & $rng.start & ", " & $rng.stop & ", " & $rng.step & ")"
   else:
-    "range($1, $2)".format(rng.start, rng.stop)
+    "range(" & $rng.start & ", " & $rng.stop & ")"
```
Comment (Yardanico) [inline src/pylib/xrange.nim:12]: Same as above

**Comment by Yardanico** [comment]
Also I wonder - do we really need to name it `xrange`? Maybe it's better to name it something like `pyrange` so it's not confused with Python 2's `xrange`? What do you think?

**Comment by juancarlospaco** [comment]
What do you mean?, it suppose to mimic `xrange`.
`xrange` was just `range` but faster because C.

**Comment by juancarlospaco** [comment]
It should require at least v1.2 ? `requires "nim >= 1.0.0"` :arrow_right: `requires "nim >= 1.2.0"` ?

**Comment by Yardanico** [comment]
> What do you mean?, it suppose to mimic `xrange`.
> `xrange` was just `range` but faster because C.
> 

Ah right, I forgot, sorry

**Comment by Yardanico** [comment]
> It should require at least v1.2 ? `requires "nim >= 1.0.0"` :arrow_right: `requires "nim >= 1.2.0"` ?

Yeah you can change that and merge

**Pull request merged**